server/sports/sports.service.ts

```diff
@@ -34,25 +35,18 @@ export default class SportsService {
 		return sports
```

Comment:
Hm, this can't be decorated with the new `Cacheable(getCachedSportsList)` decorator? (I mean this whole method)

Reply:
Well for now `Cacheable` only works with `Cache` interface, but `getCachedSportsList` and `cacheSportsList` are custom functions. 

For example, `cacheSportsList` also caches all events as well, and all of that seemed too custom and single-use to be done through decorators.

Reply:
Right, thanks for the explanation!